exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/TransferTxTest.java

```diff
@@ -59,36 +63,64 @@
   public void fromMessage() {
     long seed = 1;
     long amount = 50L;
-    BinaryMessage m = new Message.Builder()
-        .setServiceId(CryptocurrencyService.ID)
-        .setMessageType(TransferTx.ID)
-        .setBody(ByteBuffer.wrap(TxMessagesProtos.TransferTx.newBuilder()
-            .setSeed(seed)
-            .setFromWallet(fromPublicKey(fromKey))
-            .setToWallet(fromPublicKey(toKey))
-            .setSum(amount)
-            .build()
-            .toByteArray()))
-        .setSignature(new byte[Message.SIGNATURE_SIZE])
-        .buildRaw();
+    BinaryMessage m = createUnsignedMessage(seed, fromKey, toKey, amount);;
```

Comment:
Double `;`

Reply:
Thanks, updated! Actually, there is a local branch with extra clean-ups to cryptocurrency, that are better to review separately.